src/test/java/com/saralein/server/router/ServerRouterTest.java

```diff
@@ -28,17 +28,23 @@ public void setUp() {
         Path root = Paths.get(rootPath);
         SysFileHelper fileHelper = new SysFileHelper(root);
 
-        String nullString = "GET /snarf.jpg HTTP/1.1";
-        RequestParser nullParser = new RequestParser();
-        notFoundRequest = nullParser.parse(nullString);
-
-        String directoryString = "GET / HTTP/1.1";
-        RequestParser directoryParser = new RequestParser();
-        directoryRequest = directoryParser.parse(directoryString);
-
-        String fileString = "GET /cheetara.jpg HTTP/1.1";
-        RequestParser fileParser = new RequestParser();
-        fileRequest = fileParser.parse(fileString);
+        notFoundRequest = new Request(new HashMap<String, String>(){{
+            put("method", "GET");
+            put("uri", "/snarf.jpg");
+            put("version", "HTTP/1.1");
+        }});
+
+        directoryRequest = new Request(new HashMap<String, String>(){{
+            put("method", "GET");
+            put("uri", "/");
+            put("version", "HTTP/1.1");
+        }});
+
+        fileRequest = new Request(new HashMap<String, String>(){{
+            put("method", "GET");
+            put("uri", "/cheetara.jpg");
+            put("version", "HTTP/1.1");
+        }});
```

Comment:
Looks like each of these requests are individually only used once, we should just initialize them in the specific test that they're required.